app/helpers/shopping_lists_helper.rb

```diff
@@ -21,16 +19,14 @@ def display_item(item)
     # rubocop enable: Rails/OutputSafety
   end
 
-  def display_status(item)
-    # "<span class='material-icons in-cart'>shopping_cart</span>"
-    tag = <<~TAG
-      <span class='status-tag js-remove-from-cart'>
-        <span class='material-icons-outlined in-cart' title='Item is purchased and scheduled for home delivery'>
-          shopping_cart
-        </span>
-      </span>
-    TAG
-    tag.html_safe if item.in_cart?
```

Comment:
Hooray for content tags! Now I don't have to do janky stuff and use `html_safe`!